### PR TITLE
R6 user login

### PR DESCRIPTION
**Pull request by DeeDoubleYous**
Includes the current implementation of the user login. CSS styling needed to be complete, but functionally should be complete. After this save for later can be done. 
Issues with this would be with storing passwords in plain text and not doing any fancy encryption at all but in this "fake it" scenario we can  live with it.

### CODE REVIEW

File: src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java [hunk 1]
```diff
@@ -30,6 +31,8 @@
     private double totalValue = 0;
     private int totalItems = 0;
     
+    private Customer customer;
```
Comment (mc1098) [inline src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java]: Why do you need Customer here?

File: src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java [hunk 2]
```diff
@@ -130,4 +133,12 @@ public void clear(){
...
         items.clear();
     }
+
+	public Customer getCustomer() {
+		return customer;
+	}
+
+	public void setCustomer(Customer customer) {
+		this.customer = customer;
+	}
```
Comment (mc1098) [inline src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java]: As above

File: src/main/webapp/index.xhtml [hunk 3]
```diff
@@ -14,6 +14,7 @@
     </h:head>
     <h:body>
         
+        <h:outputText styleClass="log-out" value="#{LocalTradeList.customerId}"/>
```
Comment (mc1098) [inline src/main/webapp/index.xhtml]: This renders an empty span element for me above the menu. 
Would this be better in the menu component? could go in the drop-down menu after "trades"?

File: src/main/webapp/resources/css/style.css [hunk 4]
```diff
@@ -213,9 +213,6 @@ main{
     opacity: 0.7;
 }
 
-.user-sec{
-    color: #fff;
-}
```
Comment (mc1098) [inline src/main/webapp/resources/css/style.css:218]: Removing this makes the user name in the menu black.

**Review by mc1098 — CHANGES_REQUESTED** [review]
Just needs some minor changes - inputSecret is a good change. 
Sorry for delay in reviewing this.

File: src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java (hunk 1 above)
Comment (DeeDoubleYous) [reply src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java]: Oh that was for something I was going to attempt. But it's not needed now. Will get rid of when I get home.

File: src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java (hunk 2 above)
Comment (DeeDoubleYous) [reply src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java]: Same as above

File: src/main/webapp/resources/css/style.css (hunk 4 above)
Comment (DeeDoubleYous) [reply src/main/webapp/resources/css/style.css:218]: I think that maybe a victim of when I was merging your stuff into this branch. Not intended to be removed. Will reinstate.

File: src/main/webapp/resources/comp/menu.xhtml [hunk 5]
```diff
@@ -19,9 +21,15 @@
...
                     </div>
-                    <div class="user-menu">
+                    <h:form class="user-menu">
                         <a href="./trades.xhtml">Trades</a>
-                    </div>
+                        <h:commandLink id="log-out"
+                        				class="log-out-btn"
+                        				value="Log out"
+                        				action="#{loginControl.logOut(customer)}">
+                        	<f:ajax execute="@menu" resetValues="true" render="@menu"/>
```
Comment (mc1098) [inline src/main/webapp/resources/comp/menu.xhtml]: Do you need ajax here? the loginControl::logOut function is going to redirect the page anyway correct?

**Review by mc1098 — COMMENTED** [review]
Just the ajax thing I'm not sure about, does it cause an issue without it?

File: src/main/webapp/resources/comp/menu.xhtml (hunk 5 above)
Comment (DeeDoubleYous) [reply src/main/webapp/resources/comp/menu.xhtml]: it was from the experimentation previously. Its my bad habit to leave that sort of stuff in there

**Review by mc1098 — APPROVED** [review]

**Pull request merged**